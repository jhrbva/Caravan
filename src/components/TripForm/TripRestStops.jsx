import React from 'react';
import { Field, Form } from 'formik';
import '../../utilities/fonts.scss';
import Input from '../Input/Input';

const TripRestStops = () => (
	<Form>
		<Field
			icon='location_on'
			type='text'
			name='rest_stops'
			placeholder='Rest Stop'
			autoFocus={true}
			component={Input}
		/>
	</Form>
);

export default TripRestStops;
