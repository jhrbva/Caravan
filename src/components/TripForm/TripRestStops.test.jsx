import React from 'react';
import { shallow } from 'enzyme';
import TripRestStops from './TripRestStops';

describe('TripRestStops', () => {
	it('should match the snapshot', () => {
		const wrapper = shallow(<TripRestStops />);
		expect(wrapper).toMatchSnapshot();
	});
	it('should have one field', () => {
		const wrapper = shallow(<TripRestStops />);
		const fieldWrapper = wrapper.find('Field');
		expect(fieldWrapper.length).toBe(1);
	});
	it('should have one field', () => {
		const wrapper = shallow(<TripRestStops />);
		const fieldWrapper = wrapper.find('Field');
		const restStopField = fieldWrapper.find({ name: 'rest_stops' });
		expect(restStopField.length).toBe(1);
	});
});
