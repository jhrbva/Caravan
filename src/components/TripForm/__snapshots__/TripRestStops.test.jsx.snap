// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TripRestStops should match the snapshot 1`] = `
<Form>
  <Field
    autoFocus={true}
    component={[Function]}
    icon="location_on"
    name="rest_stops"
    placeholder="Rest Stop"
    type="text"
  />
  <BigButton
    value="Back"
  />
  <BigButton
    value="Submit"
  />
</Form>
`;
